docs/install.md

```diff
@@ -6,18 +6,20 @@ You need Python 3.9 or above in your working environment to use Ragna.
 
 ## Install Ragna
 
-You can install Ragna with `pip`:
+You can install Ragna and all recommended dependencies with:
 
 ```bash
-pip install ragna
+pip install 'ragna[builtin]'
 ```
 
-If you want to install all optional dependencies, append `[builtin]`:
+If you want to install a minimal version[^1]:
 
 ```bash
-pip install 'ragna[builtin]'
+pip install ragna
 ```
 
+[^1]: The minimal version is for users interested in writing plugins to extend Ragna.
```

Comment:
You can do that with the builtin installation as well. If we want a footnote here, it should be something like "the minimal version is for users that want fine-grained control over the dependencies needed for the builtin components"